common/objectconfig.hpp

```diff
@@ -41,7 +41,7 @@ class ObjectConfig : public Loggable {
 	GeographicPositionType getOrigin() const { return origin; }
 	std::string getProjString() const;
 	ATOS::Trajectory getTrajectory() const { return trajectory; }
-	void setTrajectory(const ATOS::Trajectory& newTraj) { trajectory = newTraj; } // TODO danger danger - don't do this
+	void setTrajectory(const ATOS::Trajectory& newTraj) { trajectory = newTraj; }
```

Comment:
Why was the comment removed?

Reply:
I added a copy assignment operator, but maybe this doesn't fix all the problems? Should we keep the comment?